localstack/services/lambda_/invocation/lambda_service.py

```diff
@@ -91,6 +91,7 @@ def stop(self) -> None:
         shutdown_futures = []
         for event_manager in self.event_managers.values():
             shutdown_futures.append(self.task_executor.submit(event_manager.stop))
+        # TODO: switch shutdown order?
```

Comment:
good catch 🕵️ 

In a follow-up,  I think it makes sense to start shutting down from:
1. source of invocations (i.e., prevent new invokes)
2. starting (i.e., prevent starting ones from getting running)
3. running (i.e., abort currently running ones)

Otherwise, we could miss some function versions that just transitioned.